dbt/macros/.sqlfluff

```diff
@@ -1,7 +1,7 @@
 # These rules are required to support sqlfmt. See:
 # https://docs.sqlfmt.com/integrations/sqlfluff
 [sqlfluff]
-exclude_rules = layout.indent, layout.cte_bracket, layout.select_targets, ambiguous.column_count, structure.column_order, RF04, ST05
+exclude_rules = layout.indent, layout.cte_bracket, layout.select_targets, layout.end_of_file, ambiguous.column_count, structure.column_order, RF04, ST05
```

Comment:
I'm not sure why this started failing now, but it seems like sqlfluff and sqlfmt disagree on whether there should be a trailing newline at the end of files. Even more mysteriously, sqlfluff also only seems upset about `dbt/macros/insert_hyphens.sql`, even though no other SQL files end with a trailing newline. I think the root fix here is probably to upgrade both of these dependencies, because I suspect this is a bug in one of the packages, but upgrading causes a huge diff so we should do it in a follow-up PR.